src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11Key.java

```diff
@@ -480,6 +498,28 @@ byte[] getEncodedInternal() {
         }
     }
 
+    private static class P11SecretKeyFIPS extends P11Key implements SecretKey {
+        private static final long serialVersionUID = -7828241727014329084L;
```

Comment:
Don't reuse the same UID; add missing `@Serial` annotation.

Reply:
`@Serial` is added, and what value do you suggest that I should use for this P11SecretKeyFIPS class?

Reply:
I suggest you use `serialver` from the jdk you build.

Reply:
For me, it suggested:
```java
        @Serial
        private static final long serialVersionUID = -9186806495402041696L;
```

Reply:
Updated.